msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md

```diff
@@ -1,54 +1,56 @@
 
 For more information about the organization-wide application permissions that can be granted to your app, see [online meeting permissions](/graph/permissions-reference#online-meetings-permissions).
 
 ### Use meeting-specific RSC application permissions
 
-If you want your app to fetch transcripts only for the Teams meeting where it's installed, configure  meeting-specific RSC permission for your app. Authorized users can install your app in the meeting chat. After the meeting ends, your app can make the API call to obtain the transcript for that meeting.
+If you want your app to fetch transcripts and recordings only for the Teams meeting where it's installed, configure  meeting-specific RSC permission for your app. Authorized users can install your app in the meeting chat. After the meeting ends, your app can make the API call to obtain the transcript and recording for that meeting.
 
 For more information about the meeting-specific RSC permissions, see [RSC permissions for a chat or meeting](../rsc/resource-specific-consent.md#rsc-permissions-for-a-chat-or-meeting).
 
-After you've configured the permissions, configure your app to receive change notifications for all relevant meeting events. Notifications contain meeting ID and organizer ID that help in accessing transcript content. Your app can fetch the transcript for a meeting when it's generated after it ends. The content of the transcript is available as `.vtt` or `.docx` file.
-
-> [!NOTE]
-> The docx format for transcripts will be deprecated starting May 31, 2023. For more information, see [Get callTranscript](/graph/api/calltranscript-get).
+After you've configured the permissions, configure your app to receive [change notifications](/graph/transcripts) for [transcripts available events](/graph/transcripts) or all relevant meeting events. Notifications contain meeting ID and organizer ID that help in accessing transcript content and recording. Your app can fetch the transcript and recording for a meeting when it's generated after it ends. The content of the transcript is available as `.vtt` file. The recording of the meeting is available as an `.mp4` file.
 
 For more information about how your app can know when the meetings ends, see [subscribe to change notifications](fetch-id.md#subscribe-to-change-notifications) and [use Bot Framework to get meeting ID and organizer ID](fetch-id.md#use-bot-framework-to-get-meeting-id-and-organizer-id).
 
+For more information about how your app can know when a transcript or recording is available after the online meeting ends, see [get notified when transcripts is available](/graph/transcripts).
```

Comment:
Please change to 'get notified when transcripts and recordings are available'

Reply:
Updated as suggested